src/quantum/subbenchmarks.jsx

```diff
@@ -78,7 +79,7 @@ class PerfherderContainer extends Component {
     const { perfherderUrl, data, parentSignature } = await getData({
       suite,
       platform,
-      framework,
+      framework: Number.parseInt(framework, 10),
```

Comment:
this is the only important line; the `framework` is arriving from the URL as a string "10", not a number.